content/commands/info/index.md

```diff
@@ -493,10 +493,56 @@ The **cluster** section currently only contains a unique field:
 
 *   `cluster_enabled`: Indicate Redis cluster is enabled
 
-The **modules** section contains additional information about loaded modules if the modules provide it. The field part of properties lines in this section is always prefixed with the module's name.
-
-The **keyspace** section provides statistics on the main dictionary of each
-database.
+The **modules** section contains additional information about loaded modules if the modules provide it. The field part of property lines in this section are always prefixed with the module's name.
+
+*   `search_number_of_indexes`: The total number of indexes in the shard.
+*   `search_number_of_active_indexes`: The total number of indexes running a background indexing and/or background query processing operation. Background indexing refers to vector ingestion process, or in-progress background indexer.
+*   `search_number_of_active_indexes_running_queries`: The total count of indexes currently running a background query process.
+*   `search_number_of_active_indexes_indexing`: The total count of indexes currently undergoing a background indexing process. Background indexing refers to a vector ingestion process, or an in-progress background indexer. This metric is Limited by the number of WORKER threads allocated for writing operations plus the number of indexes.
+*   `search_total_active_write_threads`: The total count of background write (indexing) processes currently running in the shard. Background indexing refers to a vector ingestion process, or an in-progress background indexer. This metric is Limited by the number of threads allocated for writing operations.
```

Comment:
Is "Limited" capitalized for a specific reason here? If so, please disregard this suggestion.
```suggestion
*   `search_number_of_active_indexes_indexing`: The total count of indexes currently undergoing a background indexing process. Background indexing refers to a vector ingestion process, or an in-progress background indexer. This metric is limited by the number of WORKER threads allocated for writing operations plus the number of indexes.
*   `search_total_active_write_threads`: The total count of background write (indexing) processes currently running in the shard. Background indexing refers to a vector ingestion process, or an in-progress background indexer. This metric is limited by the number of threads allocated for writing operations.
```